recipes/pygame/patches/0007-mark-time-tests-as-broken.patch

```diff
@@ -0,0 +1,20 @@
+From 64db68a25b6678a93339eb5282df25c96f753792 Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Fri, 29 Jul 2022 10:44:32 +0200
+Subject: [PATCH 07/10] mark time tests as broken
+
+---
+ test/time_tags.py | 1 +
+ 1 file changed, 1 insertion(+)
+ create mode 100644 test/time_tags.py
+
+diff --git a/test/time_tags.py b/test/time_tags.py
+new file mode 100644
+index 00000000..20d724a2
+--- /dev/null
++++ b/test/time_tags.py
+@@ -0,0 +1 @@
++__tags__ = ["broken"]
+-- 
+2.37.0.windows.1
```

Comment:
timing tests are broken in general on CI (something having to do with VMs having really bad clock thingies). On the pygame side we always skip them (and other things problematic on CI) by passing `--exclude opengl,music,timing` to the test command